charts/redpanda/values.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	certmanagerv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
-	cmmeta "github.com/cert-manager/cert-manager/pkg/apis/meta/v1"
+	cmmetav1 "github.com/cert-manager/cert-manager/pkg/apis/meta/v1"
```

Comment:
nit: This diverges from the alias used in certs.go. My preference is this one to match the style of other k8s imports (despite the need for an extra `cm`). This is what's causing the genpartials to waffle between two aliases.

Reply:
I'll fix this up. (done)